src/DssSpell.t.sol

```diff
@@ -36,27 +36,64 @@ contract DssSpellTest is DssSpellTestBase {
         checkCollateralValues(afterSpell);
     }
 
-//    function testCollateralIntegrations() public {
-//        vote(address(spell));
-//        scheduleWaitAndCast(address(spell));
-//        assertTrue(spell.done());
-//
-//        // Insert new collateral tests here
-//    }
-
-//    function testNewChainlogValues() public {
-//        vote(address(spell));
-//        scheduleWaitAndCast(address(spell));
-//        assertTrue(spell.done());
-//    }
-
-//    function testNewIlkRegistryValues() public {
-//        vote(address(spell));
-//        scheduleWaitAndCast(address(spell));
-//        assertTrue(spell.done());
-//
-//        IlkRegistryAbstract ilkRegistry = IlkRegistryAbstract(addr.addr("ILK_REGISTRY"));
-//    }
+    function testCollateralIntegrations() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        // Insert new collateral tests here
+        checkIlkIntegration(
+            "WBTC-B",
+            GemJoinAbstract(addr.addr("MCD_JOIN_WBTC_B")),
+            ClipAbstract(addr.addr("MCD_CLIP_WBTC_B")),
+            addr.addr("PIP_WBTC"),
+            true,
+            true,
+            false
+        );
+    }
+
+    function testNewChainlogValues() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        // Insert new chainlog tests here
+        assertEq(chainLog.getAddress("WBTC"), addr.addr("WBTC"));
+        assertEq(chainLog.getAddress("PIP_WBTC"), addr.addr("PIP_WBTC"));
```

Comment:
These aren't actually new chainlog values.

Reply:
Done in 716865f.